.github/workflows/helm-vib-lint.yaml

```diff
@@ -0,0 +1,23 @@
+name: Lint Helm Chart
+on:
+  workflow_dispatch:
+  pull_request_target:
```

Comment:
```suggestion
  pull_request_target:
    branches:
      - main
      - bitnami-labs:main
```

This is how it was before and I think it is fine. Only run on PRs whose target is `main` or `bitnami-labs:main`

Reply:
ok